audiocall/ConnectionService.MD

```diff
@@ -18,72 +18,70 @@ AndroidManifest.xml must declare a `ConnectionService` service. The telecom subs
+You can also use additional features such as putting a call on hold or muting with `connectionCapabilities`. Find the full list of features on the [Android Developers](https://developer.android.com/reference/android/telecom/Connection) site.
 
-Please find the details for creating a new Connection for outgoing and incoming VoIP calls below.
+### Make an outgoing call
+To create an outgoing call, check that a `PhoneAccount` is registered and then call the `TelecomManager.placeCall()` method.
 
-### Make outgoing call
-To create an outgoing call, it is recommended to check that a `PhoneAccount` is registered and then invoke `TelecomManager.placeCall()` method.
+A self-managed `ConnectionService` must contain `EXTRA_PHONE_ACCOUNT_HANDLE` in the Bundle extra to specify the `PhoneAccountHandle` associated with it.
 
-A self-managed `ConnectionService` must add `EXTRA_PHONE_ACCOUNT_HANDLE` to the Bundle extra to specify its associated `PhoneAccountHandle`.
+If the app can place an outgoing call, the telecom subsystem calls the `onCreateOutgoingConnection()` method to where the app must return a new `Connection` to represent a new outgoing call.
 
-If the app is able to place an outgoing call, the telecom subsystem invokes the `onCreateOutgoingConnection()` method where the app must return a new `Connection` to represent a new outgoing call.
-
-It is important to call `Connection.setActive()` once the outgoing call is connected. This will notify the telecom subsystem that the call is in progress.
+It is important to call `Connection.setActive()` once the outgoing call is connected. It notifies the telecom subsystem that the call is in progress.
 
 <img src="../screenshots/schemes/connection_scheme_outgoing_call.png">
 
-### Receive incoming call
-When the app receives an incoming call via Voximplant SDK (`IClientIncomingCallListener.onIncomingCall()`), it is necessary to check the `PhoneAccount` registration in the system. It is required to invoke the `TelecomManager.addNewinsomingCall()` method to register a new incoming call.
+### Receive an incoming call
+When the app receives an incoming call via Voximplant SDK (`IClientIncomingCallListener.onIncomingCall()`), it is necessary to check the `PhoneAccount` registration in the system. To register a new incoming call, use the `TelecomManager.addNewinsomingCall()` method.
 
-The telecom subsystem binds to the app's `ConnectionService` and requests a new `Connection` instance for the incoming call by invoking `ConnectionService.onCreateIncomingConnection()` API. Upon receiving this event, the application needs to create a `Connection`.
+The telecom subsystem connects to the app's `ConnectionService` and requests a new `Connection` instance for the incoming call by means of the `ConnectionService.onCreateIncomingConnection()` API. Upon receiving this event, the application needs to create a `Connection`.
```

Comment:
`binds` is better in this context.

From Android documentation:
> System managed ConnectionServices must be enabled by the user in the phone app settings before Telecom will bind to them. Self-managed ConnectionServices must be granted the appropriate permission before Telecom will bind to them.

Reply:
In this case, you should say "is bound to", otherwise, change it to "connects"